jobs/atc/spec

```diff
@@ -507,3 +507,42 @@ properties:
     description: |
       Client secret for CredHub authorization.
     default: ""
+
+  ssm.aws_region:
+    description: |
+      AWS region to use for fetching SSM parameters. Enviroment variable AWS_REGION is used
+      if this flag is not provided. This flag or the environment variable *must* be specified
+      for SSM secret backend to be enabled. When AWS_SDK_LOAD_CONFIG environment variable is
+      set, shared configuration files (~/.aws/config & ~/.aws/credentials) are used to load
+      this flag and credential flags. A profile within shared config file may be selected by
+      specifying it in AWS_PROFILE or AWS_DEFAULT_PROFILE environment variables.
+    default: ""
```

Comment:
These defaults can be removed since you're using `if_p` anyway, and they're not really defaults. :)